### PR TITLE
WIP: Fix zip extraction issue

### PR DESCRIPTION
**Pull request by rgolangh**
Signed-off-by: Roy Golan <rgolan@redhat.com>

### CODE REVIEW

**Comment by rgolangh** [comment]
@gabriel-farache @masayag building on the recomendation of snyk this could potentially fix it.

**Comment by rgolangh** [comment]
hopefully konflux pipeline would check it for us

File: e2e/resources/move2kube-instance.yaml [hunk 1]
```diff
@@ -36,7 +42,19 @@ spec:
             postStart:
               exec:
                 command: [ "/bin/sh", "-c", "ssh-agent -a /tmp/unix-socket && ssh-add /root/.ssh/id_rsa && ssh git@github.com -o StrictHostKeyChecking=accept-new || true && ssh git@gitlab.com -o StrictHostKeyChecking=accept-new|| true && ssh git@bitbucket.org -o StrictHostKeyChecking=accept-new|| true" ]
+      initContainers:
+        -   name: log-shipper
+            image: docker.io/alpine:latest
```
Comment (masayag) [inline e2e/resources/move2kube-instance.yaml:47]: perhaps a fixed version instead of latest?

**Comment by rgolangh** [comment]
I am going to close this PR and re-open it against a branch on the upstream remote and not from my fork in order to
sure the e2e runs well and have access to the workspace secret

**Pull request closed without merging**